packages/console/src/components/RadioGroup/index.module.scss

```diff
@@ -0,0 +1,68 @@
+@use '@/scss/underscore' as _;
+
+.radioGroup {
+  display: flex;
+
+  &:not(:first-child) {
+    margin-top: _.unit(3);
+  }
+
+  svg {
+    opacity: 0%;
```

Comment:
would using display: none or visibility: hidden work? as dom won't render the element or handle any of its events.

Reply:
I think `visibility="hidden"` would be the best choice, for settings on `display` and `opacity` will break the value inherited from their parent when they are checked.

Reply:
consider css animation, opacity will be better?

Reply:
not a blocker, some complicated solution might be adding transition end then set display none.  As long as we don't have any event triggering or render concerns. 

<img width="689" alt="image" src="https://user-images.githubusercontent.com/36393111/157179674-4602e8d2-24ed-4545-bbca-21fb2550b15f.png">

Reply:
feel free to resolve

Reply:
agreed - for the transition i would prefer using `react-spring` for it then we don't need to take care on this